src/Containers/Settings.tsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState } from 'react';
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import styled from 'styled-components';
+import { InfoCircle } from 'styled-icons/fa-solid/InfoCircle';
+import { Wrench } from 'styled-icons/fa-solid/Wrench';
+import { Book } from 'styled-icons/fa-solid/Book';
+import { QuestionCircle } from 'styled-icons/fa-solid/QuestionCircle';
+import { Mixins, BaseStyles } from '@Utils';
+
+import { SignOutAlt } from 'styled-icons/fa-solid/SignOutAlt';
+import { UserCircle } from 'styled-icons/fa-solid/UserCircle';
+import { Heading, Paragraph, SmallText } from '../Text';
+import { Loading } from './Loading';
+import { SettingsCard } from './SettingsCard';
+import { Button } from '../Inputs/Button';
+import { Modal } from './Modal';
+import { Copyright } from './Copyright';
+
+const loadingStyle = `
+    ${Mixins.flex('column', 'flex-start', 'stretch')}
+    ${Mixins.scroll}
+    overflow: auto;
+    width: 100%;
+`;
+
+interface SettingsProps
+    extends MainInterface,
+        ResponsiveInterface,
+        InfoProps,
+        AccountProps {
+    loading: boolean;
+    cardOne?: React.ReactElement;
+    cardTwo?: React.ReactElement;
+    cardThree?: React.ReactElement;
+}
+interface InfoProps extends MainInterface, ResponsiveInterface {
+    faqName: string;
+    faqLink: string;
+    version: string | number;
+}
+interface AccountProps extends MainInterface, ResponsiveInterface {
+    employee: { username: string; role: string; _id: string };
+    logout: Function;
+}
+interface DisplayItemProps extends MainInterface, ResponsiveInterface {
+    label: string;
+    value: string;
+}
+
+const Info: React.FC<InfoProps> = ({ faqName, faqLink, version }) => {
```

Comment:
Something i failed to do, which we need for every component to fix most of our TS issues is to inlude the props from the actual html element which the component is based off
https://stackoverflow.com/questions/58924179/typscript-react-extending-default-functional-component-props-gives-error

Reply:
So we need to include the html props for every type of element that is present in the component, or the props for the outermost element (in the code above, the outermost element is SettingsCard which is a div)?

Reply:
outmost